src/backend/replication/logical/logical.c

```diff
@@ -1236,6 +1236,8 @@ message_cb_wrapper(ReorderBuffer *cache, ReorderBufferTXN *txn,
 
 	if (ctx->callbacks.message_cb == NULL)
 		return;
+	if (strncmp(prefix, "neon-file", sizeof("neon-file") == 0)
```

Comment:
I think you are missing a -1 after the sizeof. sizeof on a string literal includes the nul byte

Reply:
Good point. We can probably just change it to strcmp honestly